Form/Step.php

```diff
@@ -62,6 +70,20 @@ public static function createFromConfig($number, array $config) {
 	}
 
 	/**
+	 * @param array $form_options
+	 */
+	public function setFormOptions($form_options) {
+		$this->form_options = $form_options;
```

Comment:
I asked you to remove the type hint only, not the whole type checking. :smirk: So please add the `if (is_array())` stuff again.